.travis.yml

```diff
@@ -287,7 +291,10 @@ jobs:
     env:
       <<: *env-functional-shard_3
       ANSIBLE_VERSION: devel
-
+  - release
+      <<: *if-cron-or-manual-run-or-tagged
```

Comment:
And cron is not a good choice. It'd trigger multiple (failing) attempts to publish the same commit/version. Unless version doesn't contain date stamp, which wouldn't make sense either because we'd end up with a number of versions with exactly same source polluting PyPI page history.

Reply:
What do you propose to use? We could check the output and ingore the failure to upload due to already-uploaded and keeping it as failed only for "other reasons".

How often does the cron runs? I think that weekly is ok, more often would create spam. Still, keep in mind that we can hide releases from pypi. We could easily auto-hide old pre-releases to declutter the history interface.

Reply:
It runs daily. I already have a good implementation and I don't see a reason to reimplement that.